geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/RegionManagementController.java

```diff
@@ -196,4 +196,20 @@ public ClusterManagementResult deleteRegion(
     return new ResponseEntity<>(result,
         HttpStatus.CREATED);
   }
+
+  @ApiOperation(value = "delete region index")
+  @PreAuthorize("@securityService.authorize('CLUSTER', 'MANAGE', 'QUERY')")
```

Comment:
would be nice to add an entry in the "ClusterManagementSecurityRestIntegrationTest" to test this.

Reply:
Added context for testing permissions in `ClusterManagementSecurityRestIntegrationTest`.